CHANGES.txt

```diff
@@ -2,6 +2,86 @@
 Pyface Changelog
 ================
 
+Release 7.1.0
+=============
+
+Released: 2020-10-XX
+
+Pyface 7.1.0 introduces a few new and major features and fixes to deprecation
+warnings. There are also internal cleanups to Qt signal connection logic that
+should be invisible to users.
+
+This release should be compatible with Traits 6.0+. Users are encouraged to
+upgrade to Traits 6.1+ to stay current as future releases of Pyface will stop
+supporting Traits 6.0.
+
+Highlights of this release
+--------------------------
+
+* A new :mod:`data_view <pyface.data_view>` subpackage has been introduced to
+  support visualization of hierarchical and non-hierarchical tabular data. See
+  :ref:`data-view` section of the user manual for an introduction. Additional
+  examples are also added to the source distribution. Please note that as of
+  this release the API is provisional.
+
+* A new :class:`Color <pyface.color.Color>` class and a corresponding
+  :class:`PyfaceColor <pyface.ui_traits.PyfaceColor>` trait type have been
+  introduced to support representing colors via a toolkit independent API. See
+  :ref:`ui-trait-types` section of the user manual for further details.
+
+Detailed changes
+----------------
+
+More than 70 PRs went into this release. Thanks to:
+
+* Aaron Ayres
+* Ieva Cernyte
+* Mark Dickinson
+* Kit Yan Choi
+* Rahul Poruri
```

Comment:
@rahulporuri GitHub shows your name as "Poruri Sai Rahul", but subsequent mentions of your name in this file is "Rahul Poruri".

Reply:
either is fine. thanks for the headsup. resolving.